tutorials/misc/plot_report.py

```diff
@@ -115,21 +115,39 @@
 
 ###############################################################################
 # Now let's look at how :class:`~mne.Report` handles :class:`~mne.Evoked` data
-# (we'll skip the MRIs to save computation time):
+# (we'll skip the MRIs to save computation time). The following code will
+# produce butterfly plots, topomaps, and comparisons of the global field
+# power (GFP) for different experimental conditions.
 
 pattern = 'sample_audvis-no-filter-ave.fif'
 report = mne.Report(verbose=True)
 report.parse_folder(path, pattern=pattern, render_bem=False)
 report.save('report_evoked.html')
 
 ###############################################################################
-# To render whitened :class:`~mne.Evoked` files with baseline correction, add
-# the noise covariance file. This will display ERP/ERF plots for both the
-# original and whitened :class:`~mne.Evoked` objects, but scalp topomaps only
-# for the original.
+# You have probably noticed that the EEG recordings look particularly odd. This
+# is because by default, `~mne.Report` does not apply baseline correction
+# before rendering evoked data. To request baseline correction, pass a
```

Comment:
It's both a combination of how the Report works and how the dataset was created, i.e.:

"... evoked data, and the MNE sample evoked dataset was created without applying baseline correction."

Reply:
Right, the phrasing was a little unfortunate there. I tried to address this in 4248ff7, WDYT?